src/v2/server.py

```diff
@@ -0,0 +1,71 @@
+from gi.repository import Gio
+from gi.repository import GLib
+from gi.repository import HackSound
+
+from hack_sound_server.server import Server as _Server
+from hack_sound_server.v2.dbus.manager import DBusManager
+from hack_sound_server.v2.player import Player
+from hack_sound_server.v2.sound import ServerSoundFactory
+
+
+class ServerManager(DBusManager):
+    _SKELETON_CLASS = HackSound.Server2Skeleton
+
+    def __init__(self, server):
+        super().__init__(server, server)
+        self._owner_id = None
+
+    @property
+    def interface_name(self):
+        return "com.endlessm.HackSoundServer2"
+
+    @property
+    def object_path(self):
+        return "/com/endlessm/HackSoundServer2"
+
+    def get_player(self, invocation, app_id, unused_options):
+        player = self.target_object.registry.players_by_bus_name.get(app_id)
+        if not player:
+            player = Player(self.target_object, app_id)
+            self.target_object.registry.players_by_bus_name[app_id] = player
+        invocation.return_value(GLib.Variant("(o)", (player.object_path, )))
+
+    def register_object(self, unused_connection=None, path=None):
```

Comment:
I do not know how can the server application can own two names. The only way that came to my mind was to wait until the interface "com.endlessm.HackSoundServer" gets registered to also register "com.endlessm.HackSoundServer2". But I think that is limiting autoinitialization. Not sure what is the best way to do it.